kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/containExactlyInAnyOrder.kt

```diff
@@ -147,3 +153,22 @@ fun <T, C : Collection<T>> containExactlyInAnyOrder(
       negatedFailureMessage
    )
 }
+
+internal fun<T> countMismatch(expectedCounts: Map<T, Int>, actualCounts: Map<T, Int>) =
```

Comment:
Does this work for `List<T?>`? Is there a nested nullability issue?

Reply:
> Does this work for `List<T?>`? Is there a nested nullability issue?

it does work with nulls, added tests to verify that